src/common/api/customBaseQuery.ts

```diff
@@ -10,9 +10,11 @@ const baseQuery = fetchBaseQuery({
   baseUrl: environment.apiRoute,
   prepareHeaders: (headers: Headers, { getState }) => {
     const { token } = (getState() as RootState).auth;
+    const currentLanguage = localStorage.getItem('language');
 
-    if (token) {
+    if (token && currentLanguage) {
       headers.set('authorization', `Token ${token}`);
+      headers.set('accept-language', currentLanguage);
     }
```

Comment:
These should probably be separate. We still want to set the token, even if currentLanguage is not set for some reason.